include/ziti/ziti_tunnel_cbs.h

```diff
@@ -13,6 +13,40 @@ XX(data, string, map, data, __VA_ARGS__)
 
 DECLARE_MODEL(tunneler_app_data, TUNNELER_APP_DATA_MODEL)
 
+#define TUNNEL_COMMANDS(XX,...) \
+XX(ZitiDump, __VA_ARGS__)    \
+XX(LoadIdentity, __VA_ARGS__)   \
+XX(ListIdentities, __VA_ARGS__)
+
+DECLARE_ENUM(TunnelCommand, TUNNEL_COMMANDS)
+
+#define TUNNEL_CMD(XX, ...) \
+XX(command, TunnelCommand, none, command, __VA_ARGS__) \
+XX(data, json, none, data, __VA_ARGS__)
+
+#define TUNNEL_CMD_RES(XX, ...) \
+XX(success, bool, none, success, __VA_ARGS__) \
+XX(error, string, none, error, __VA_ARGS__)\
+XX(data, json, none, data, __VA_ARGS__)
+
+#define LOAD_IDENTITY(XX, ...) \
+XX(path, string, none, path, __VA_ARGS__)
+
+#define IDENTITY_INFO(XX, ...) \
```

Comment:
Are you opposed to naming these so that it's clear they are tunnel commands? e.g. `TUNNEL_CMD_IDENTITY_INFO`, `TUNNEL_CMD_IDENTITY_LIST`?